java/debugger.jpda/src/org/netbeans/modules/debugger/jpda/models/ShortenedStrings.java

```diff
@@ -472,4 +630,20 @@ private static class StringValueInfo {
             this.shortValueRef = new WeakReference<String>(shortenedValue);
         }
     }
+    private enum InternalStringEncoding {
```

Comment:
Please also make sure, that there is an empty line between the end of the previous method and the enum declaration.